package/libyui-gtk.spec

```diff
@@ -84,8 +84,8 @@ This package has very few dependencies.
 
 %build
 
-export CFLAGS="$RPM_OPT_FLAGS -DNDEBUG -Wno-error=deprecated-declarations"
-export CXXFLAGS="$RPM_OPT_FLAGS -DNDEBUG -Wno-error=deprecated-declarations"
+export CFLAGS="$RPM_OPT_FLAGS -DNDEBUG -Wno-error=deprecated-declarations -fPIC"
+export CXXFLAGS="$RPM_OPT_FLAGS -DNDEBUG -Wno-error=deprecated-declarations -fPIC"
```

Comment:
Interestingly it fails only on x86_64, on i586 it's fine... I guess it might be needed because of gtk3, that's why the other packages do not need that.

Anyway, let's simply keep it this way...